src/render/index.html.eco

```diff
@@ -13,65 +13,109 @@ layout: default
+        <div class="col-sm-2">
+          <h2>Experience Manageable Integrations + Continuous Replatforming</h2>
+          <p>
+            combines various systems in a controlled and isolated way, preventing any
+            undesired interferences; gradually embed new systems, CMSes on your website
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Resilience</h2>
+          <p>
+            keeps your platform always available. Even if a module breaks, it can redeploy
+            themselves into a new location/server
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Microservices 2.0</h2>
+          <p>
+            fine-grained modular integration logic (modules with precise responsibility) with
+            manageable contracts, allowing them to be scaled separately without affecting the
+            codebase
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Scaling</h2>
+          <p>
+            integration logic can be precisely scaled vertically and horizontally, the platform
+            provides load balancing for free (the platform implements reactive principles)
+          </p>
+        </div>
+        <div class="col-sm-1">
+          <h2>Extendable & Polyglot</h2>
+          <p>
+            extension points supporting multiple programming languages
+          </p>
+        </div>
+        <div class="col-sm-2">
+          <h2>Channels / Users</h2>
+          <p>
+            the IoT is coming, modern platforms have to support both different channels like tablets,
+            mobiles, computers and other applications. With Knot.x you can address the need of
+            any channel, SPA, etc.
+          </p>
+        </div>
+        <div class="row">
+          <div class="col-sm-12">
+            <img src="img/knotx-platform-overview.png" class="img-responsive img-centered" alt="Knot.x Platform Overview"/>
+          </div>
         </div>
       </div>
     </div>
   </div>
-  <!-- Features -->
-  <div class="feature gray-background">
+
+
+  <!-- Manageable Integrations -->
+  <div class="feature white-background">
     <div class="container">
       <div class="row">
         <div class="col-sm-6">
-          <h2>Features</h2>
+          <h2>Manageable Integrations</h2>
           <p>
-            Knot.x assembles all features/services into HTML pages in a very efficient manner.
-            It loads and analyses static pages from Repository, collects dynamic features
-            asynchronously and injects them into your page.
+            Knot.x can combine experiences of the legacy CMS and the Adobe Experience Manager (or
+            other modern CMS) together with a dynamic data from CRM. The integration can be gradually
+            introduced, first by integrating AEM driven components, then dynamic components from CRM.
+            Knot.x allows to integrate with all kind of systems from CRMs via custom web-services to
```

Comment:
_Knot.x integrates with all kinds of systems from CMS, through web services, to low-level storages like databases or caches and it's not limited to HTTP_

Reply:
What about:
>Knot.x can combine the experiences of a legacy CMS and a modern one (such as the Adobe
            Experience Manager) while your site is undergoing a migration. Moreover it connects the 
            authoring experience that CMS gives with the dynamic data from external systems (such as 
            CRM, stock exchange service). So the integration can be gradually introduced, first by 
            integrating AEM-driven components, then dynamic components from various systems like CRM.
            Knot.x integrates with all kinds of systems from CMS, through web services, to low-level
            storages like databases or caches and it's not limited to HTTP.